cmake/macros/macro_expand_instantiations.cmake

```diff
@@ -78,7 +78,12 @@ MACRO(EXPAND_INSTANTIATIONS _target _inst_in_files)
   #
   FOREACH(_build ${DEAL_II_BUILD_TYPES})
     STRING(TOLOWER ${_build} _build_lowercase)
+
     ADD_DEPENDENCIES(${_target}.${_build_lowercase} ${_target}.inst)
```

Comment:
So then, out of curiosity, why do we need to do this here at all? If cmake knows how to create the `.inst` files, and if it knows about include dependencies, should it not know that it needs to generate a `.inst` file before compiling a `.cc` file that needs it?